legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -182,16 +183,23 @@ def search_businesses():
         for draft_dao in draft_query.all():
             draft = {
                 'identifier': draft_dao.temp_reg,  # Temporary registration number of the draft entity
-                'legalType': draft_dao.json_legal_type  # Legal type of the draft entity
+                'legalType': draft_dao.json_legal_type,  # Legal type of the draft entity
+                'draftType': Filing.FILINGS.get(draft_dao.filing_type, {}).get('temporaryCorpTypeCode'),
+                'draftStatus': draft_dao.status
             }
+
+            if (draft_dao.status in [Filing.Status.PAID.value, Filing.Status.APPROVED.value] and
+                    draft_dao.effective_date and draft_dao.effective_date > datetime.now(timezone.utc)):
+                draft['isFutureEffective'] = True
```

Comment:
@severinbeauvais @ArwenQin these are the new props

Reply:
Thanks.

I'm waiting to see the final designs from Andy; he might want the tooltip in the MBR to show the FED.

Reply:
Can we tell, from this data, whether the draft is pending + payment incomplete?

Reply:
if draftStatus is `PENDING` that means a payment flow is initiated (either user did not complete the payment or the system did not mark the filing as `PAID`)

Reply:
May I know what is a paid-but-not-completed filing? Will it happen?

Reply:
draftStatus will be in (`PAID` or `APPROVED`) (and isFutureEffective=True if its future effective)

Reply:
> May I know what is a paid-but-not-completed filing? Will it happen?

It could be:
1. Filer is stuck
2. filing is Future Effective

Reply:
So, yes, Andy wants to show the FED in the tooltip, so could we get that in the response also?

"This filing will become effective on Aug 10, 2024."

The UI could infer that the filing is FE if a FED is present (no need for isFutureEffective). What do you think, Vysakh and Arwen?

Reply:
> So, yes, Andy wants to show the FED in the tooltip, so could we get that in the response also?
> 
> "This filing will become effective on Aug 10, 2024."
> 
> The UI could infer that the filing is FE if a FED is present (no need for isFutureEffective). What do you think, Vysakh and Arwen?

Agree

Reply:
OK, the latest change looks good to me. I have no further comment on this conversation.